app/test/process_proposal_test.go

```diff
@@ -51,26 +50,6 @@ func TestProcessProposal(t *testing.T) {
 			testfactory.RandomBlobNamespaces(tmrand.NewRand(), 4),
 			[][]int{{100}, {1000}, {420}, {300}},
 		),
-		blobfactory.DefaultTxOpts()...,
-	)
-
-	largeMemo := strings.Repeat("a", appconsts.MaxTxSize(appconsts.LatestVersion))
```

Comment:
aren't memo's capped at 256? if so, is this a realistic test? Won't all tx with this big of a memo be rejected as is?

Reply:
I found https://github.com/celestiaorg/celestia-app/pull/3909#discussion_r1797069274

which I find incredibly opaque and like an enormous footgun

this test should still be changed imo

Reply:
the thing we're trying to prevent is blobs that are too big, therefore that's what needs to be tested. using large memo's as a proxy should be avoided

Reply:
I'm removing this test here in process proposal because it won't be removing large blobs and we should add proper testing once we fix the ante handler. wdyt?

Reply:
we definitely don't have to add tests to processproposal since we shouldn't be changing it. if anything we might want a test that confirms no breaking change was made.

I just found this test very confusing, as it was on process proposal and we're adding a memo that should be invalid by itself.

Reply:
good idea. I'll add a test with large blob and make sure it doesn't get rejected.